Documentation/CRDs/ceph-cluster-crd.md

```diff
@@ -1429,6 +1430,9 @@ In order to configure an external Ceph cluster with Rook, we need to extract som
     !!! note
         For CephFS StorageClass you also need to export `CEPHFS_FS_NAME`, `CEPHFS_POOL_NAME` or you can pass these parameters with a CLI flag (--cephfs-data-pool-name,--cephfs-filesystem-name) while running the python script. For creating ECRBDStorageClass you need to pass --rbd-metadata-ec-pool-name CLI flag or export `RBD_METADATA_EC_POOL_NAME`.
 
+    !!! note
+        For creating CephFS PVC in a subvolumeGroup and RBD PVC in a RadosNamespaces you need to pass the subvolume group name `--subvolume-group` and rados namespace name `--rados-namespace` respectively CLI flag while running the python script.
```

Comment:
Why do we need a note? Looks like these flags are already documented above on lines 1341-1342. Notes are really for pointing out important things that aren't already described in other parts of the doc.

Reply:
This is a feature added on special request,

The above is just their definition of the flags what they actually means,

Maybe we can have two sections:

1) want to create rbd and cephfs sc

2) want to create rados and subvolume sc

Reply:
Let's try to just clarify the parameter descriptions instead of use separate notes.

Reply:
DOne